src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -470,7 +469,7 @@ g.test('color_attachments,sample_count')
     }
   });
 
-g.test('resolveTarget,sample_count')
+g.test('color_attachments,resolve_target_sample_count')
```

Comment:
I personally prefer the `resolveTarget,*` naming because these are all testing validation about the `resolveTarget` member. And the name is sufficient to understand what the group of tests is testing.

Reply:
Ah, I see. Let me restore the name and change `color_attachments,resolve_target_foo` to `resolveTarget,foo`. Thanks!